ios/FluentUI.Demo/FluentUI.Demo/Demos/DrawerVnextDemoController.swift

```diff
@@ -0,0 +1,145 @@
+            ],
+            itemSpacing: Constants.verticalSpacing,
+            stretchItems: true
+        )
+        addRow(
+            items: [
+                createButton(title: "Show from leading with dimmed background", action: #selector(showLeftDrawerDimmedBackgroundButtonTapped)),
+                createButton(title: "Show from trailing with dimmed background", action: #selector(showRightDrawerDimmedBackgroundButtonTapped))
+            ],
+            itemSpacing: Constants.verticalSpacing,
+            stretchItems: true
+        )
+        addDescription(text: "Swipe from the left or right edge of the screen to reveal a drawer interactively")
+
+        container.addArrangedSubview(UIView())
+
+        // Screen edge gestures to interactively present side drawers
+
+        let leadingEdgeGesture = UIScreenEdgePanGestureRecognizer(target: self, action: #selector(handleScreenEdgePan))
+        leadingEdgeGesture.edges = view.effectiveUserInterfaceLayoutDirection == .leftToRight ? .left : .right
+        view.addGestureRecognizer(leadingEdgeGesture)
+        navigationController?.navigationController?.interactivePopGestureRecognizer?.require(toFail: leadingEdgeGesture)
+
+        let trailingEdgeGesture = UIScreenEdgePanGestureRecognizer(target: self, action: #selector(handleScreenEdgePan))
+        trailingEdgeGesture.edges = view.effectiveUserInterfaceLayoutDirection == .leftToRight ? .right : .left
+        view.addGestureRecognizer(trailingEdgeGesture)
+
+        drawerController = DrawerVnext(contentViewController: DrawerContentController())
+    }
+
+    @objc private func showLeftDrawerClearBackgroundButtonTapped() {
+        if let drawerController = drawerController {
+            drawerController.drawerState.backgroundDimmed = false
+            drawerController.drawerState.presentationDirection = .left
+            present(drawerController, animated: true, completion: nil)
+        }
+    }
+
+    @objc private func showLeftDrawerDimmedBackgroundButtonTapped() {
+        if let drawerController = drawerController {
+            drawerController.drawerState.backgroundDimmed = true
+            drawerController.drawerState.presentationDirection = .left
+            present(drawerController, animated: true, completion: nil)
+        }
+    }
+
+    @objc private func showRightDrawerClearBackgroundButtonTapped() {
+        if let drawerController = drawerController {
+            drawerController.drawerState.backgroundDimmed = false
+            drawerController.drawerState.presentationDirection = .right
+            present(drawerController, animated: true, completion: nil)
+        }
+    }
+
+    @objc private func showRightDrawerDimmedBackgroundButtonTapped() {
+        if let drawerController = drawerController {
+            drawerController.drawerState.backgroundDimmed = true
+            drawerController.drawerState.presentationDirection = .right
+            present(drawerController, animated: true, completion: nil)
```

Comment:
How would a SwiftUI application use/call the drawer in order to be displayed?

Reply:
I have added SwiftUI drawer preview code for an application using SwiftUI